process/common.go

```diff
@@ -568,3 +574,26 @@ func SortHeadersByNonce(headers []data.HeaderHandler) {
 func IsInProperRound(index int64) bool {
 	return index%RoundModulusTrigger == 0
 }
+
+// AddHeaderToBlackList adds a hash to black list handler. Logs if the operation did not succeed
+func AddHeaderToBlackList(blackListHandler BlackListHandler, hash []byte) {
```

Comment:
these logs can be done inside blackListHandler add function. no need to define in common

Reply:
TimeCache is generic. It could be used for any kind of hash (tx, miniblock, headers, etc). If I want to know the type of the object connected with the hash which has been added to the black list I have to log it from outside.